From 6cda8aa3305695133df8dfb12405670953cd8df7 Mon Sep 17 00:00:00 2001
From: Pavel Moravec <pmoravec@redhat.com>
Date: Fri, 19 Aug 2016 23:18:26 +0200
Subject: [PATCH] [omsa] add omreport storage controller

Collect "omreport storage controller".

Do not collect "omreport chassis summary" as it is invalid command.

Resolves: #868.

Signed-off-by: Charles Rose <charles_rose@dell.com>
Signed-off-by: Pavel Moravec <pmoravec@redhat.com>
---
 sos/plugins/omsa.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sos/plugins/omsa.py b/sos/plugins/omsa.py
index fa825c8..132e205 100644
--- a/sos/plugins/omsa.py
+++ b/sos/plugins/omsa.py
@@ -48,7 +48,7 @@ class omsa(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
             "{0} system events".format(self.omreport),
             "{0} chassis info".format(self.omreport),
             "{0} chassis biossetup".format(self.omreport),
-            "{0} chassis summary".format(self.omreport),
+            "{0} storage controller".format(self.omreport),
         ], timeout=30)
 
 # vim: et ts=4 sw=4
-- 
2.4.11

